geeksurvey/templates/home.html

```diff
@@ -1,40 +1,129 @@
 {% extends "base.html" %}
-
 {% load static %}
 
-{% block extra_head %}
-<style>
-  .btn {
-    display:inline-block;
-    margin:.5rem;
-  }
-  .card-img-top {
-    max-width:20%;
-  }
-</style>
+{% block base_head %}
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <meta name="description" content="" />
+  <meta name="author" content="" />
+  <title>GeekSurvey - Home</title>
+  <link rel="shortcut icon" href="{% static 'gkslogo.png' %}" type="image/x-icon">
 {% endblock %}
-
 {% block content %}
-
-{% if user.is_authenticated %}
-  <h2> GeekSurvey Home </h2>
-  <hr>
-  <p>Hi {{ user.username }}!</p>
-  <p><a href="{% url 'profile_update' %}"> Complete Your Profile </a></p>
-  <p> <a href="{% url 'participate' %}"> Find Studies </a> </p>
-  <p> <a href="{% url 'study_create' %}"> Create a Study </a> </p>
-  <hr>
-  <p> <a class="btn btn-danger" href="{% url 'account_logout' %}"> log out </a> </p>
-{% else %}
-<div class="card">
-  <div class="card-body">
-    <h1 class="card-title">GeekSurvey</h1>
-    <img src="{% static 'gkslogo.png' %}" class="card-img-top" />
-    <h4 class="card-text">A tool for facilitating software engineering research. </h4>
-    <a class="btn btn-primary" href="{% url 'account_login' %}">log in</a><br>
-    <a class="btn btn-secondary" href="{% url 'account_signup' %}">sign up</a>
-  </div>
-</div>
-{% endif %}
-
+  <!-- Logic for when a user visits the home page and is authenticated. -->
+  <!-- TODO: Update this once a template has been applied to all other pages. -->
+  {% if user.is_authenticated %}
+    <h2> GeekSurvey Home </h2>
+    <hr>
+    <p>Hi {{ user.username }}!</p>
+    <p><a href="{% url 'profile_update' %}"> Complete Your Profile </a></p>
+    <p> <a href="{% url 'participate' %}"> Find Studies </a> </p>
+    <p> <a href="{% url 'study_create' %}"> Create a Study </a> </p>
+    <hr>
+    <p> <a class="btn btn-danger" href="{% url 'account_logout' %}"> log out </a> </p>
+  <!-- Logic for when a user visits the home page and is NOT authenticated. -->
+  {% else %}
+    <!-- Business Header -->
+    <header class="bg-dark py-5">
```

Comment:
Can you put a comment here and in base.html that refers to Start Bootstrap LLC for the original source of some of the HTML?

Reply:
Looks good!